plugins/leemons-plugin-tasks/frontend/src/components/Inputs/ConditionalInput/index.js

```diff
@@ -1,6 +1,7 @@
 import React, { useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
-import { Switch, Stack, PageContainer } from '@bubbles-ui/components';
+import { Switch, Stack, PageContainer, Box } from '@bubbles-ui/components';
```

Comment:
```suggestion
import { Switch, Stack, PageContainer } from '@bubbles-ui/components';
```

Remove the unused Box

Reply:
done